### ISSUE TITLE
Require any json file that contains the word "process" is not possible

### ISSUE THREAD

**Issue by btamas** [lasso-js/lasso-require#21]
I have a project that has a json language file. It contains somewhere the word *process* as the part of a value. But when I want to require it, I cannot build the project. I got the following error:

```
WARN lasso-require/lib/inspect-cache: Error parsing JavaScript file config/languages/en.json Error: Error: Line 1: Unexpected token :
    at inspect (node_modules/gulp-lasso/node_modules/lasso/node_modules/lasso-require/lib/util/inspect.js:155:29)
    at node_modules/gulp-lasso/node_modules/lasso/node_modules/lasso-require/lib/inspect-cache.js:62:17
    at DeferredReadable.<anonymous> (node_modules/gulp-lasso/node_modules/lasso/node_modules/lasso-require/lib/util/streamToString.js:11:13)
    at DeferredReadable.emit (events.js:104:17)
    at DeferredReadable.emit (node_modules/gulp-lasso/node_modules/lasso/lib/util/DeferredReadable.js:43:37)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)

Source:
-------
{"foo": "bar process baz"}
-------

```

Maybe I found the source of the bug. In the line 9 of lib/utils/inspect.js file, there is a shourtCircuitRegExp variable that contains some reserved word, but I think it should not be check the json files.

I make a workaround and rename the json to js and insert a module.exports in it, and it's working now, but I would be happy if you could check this issue.

Thanks in advance.

**Comment by patrick-steele-idem** [comment]
That's not good. Let me investigate and I'll push out a fix ASAP. Thanks for investigating the issue and sharing your findings!

**Comment by patrick-steele-idem** [comment]
New version published: `lasso-require@1.4.7`

Thanks again, @btamas

**Comment by btamas** [comment]
@patrick-steele-idem, thank you very much for the fast bugfix :)
